index.js

```diff
@@ -131,17 +144,20 @@ cmd.lsCollaborators = (spec, user, opts) => {
     opts = user
     user = undefined
   }
-  opts = AccessConfig(opts)
-  return pwrap(opts, () => {
-    return getStream.array(
-      cmd.lsCollaborators.stream(spec, user, opts)
-    ).then(data => data.reduce((acc, [key, val]) => {
-      if (!acc) {
-        acc = {}
-      }
-      acc[key] = val
-      return acc
-    }, null))
+  return new Promise((resolve, reject) => {
+    return cmd.lsCollaborators.stream(spec, user, opts)
+      .collect()
+      .then(data => {
+        const collabList = data.reduce((acc, [key, val]) => {
+          if (!acc) {
+            acc = {}
+          }
+          acc[key] = val
+          return acc
+        }, null)
+        return resolve(collabList)
+      })
+      .catch(reject)
```

Comment:
No need to wrap in a promise.  This whole bit can be just:

```js
    return cmd.lsCollaborators.stream(spec, user, opts)
      .collect()
      .then(data => data.reduce((acc, [key, val]) => {
        if (!acc) {
          acc = {}
        }
        acc[key] = val
        return acc
      }, null))
```

Reply:
Had to wrap in a promise because `spec = npar(spec)` in `cmd.lsCollaborators.stream` threw before the return of the promise in that function.